.github/workflows/deploy-developer-docs.yml

```
@@ -0,0 +1,44 @@
+name: Deploy on Developer Docs
+
+# Run the deployment only when code is committed to teh branch.
+on:
+  push:
+    branches:
+      # Remove this once implemented and keep just `master` branch.
+      - next-release
+      - master
```

```suggestion
      - master
      - developer-doc-github-action
```

Run the action from the current branch until this is developed, This can be removed when this action is ready.

---

Okay 👍🏼